doc/langref.html.in

```diff
@@ -2929,9 +2929,15 @@ test "basic slices" {
     // Both can be accessed with the `len` field.
     var known_at_runtime_zero: usize = 0;
     const slice = array[known_at_runtime_zero..array.len];
+    try expect(@TypeOf(slice) == []i32);
     try expect(&slice[0] == &array[0]);
     try expect(slice.len == array.len);
 
+    // If you slice with comptime-known start and end positions, the result is
+    // be a pointer to an array, rather than a slice.
```

Comment:
```suggestion
    // If you slice with comptime-known start and end positions, the result is
    // a pointer to an array, rather than a slice.
```

Reply:
Lol, thank you for catching that. I was batting between "is" and "will be"